Makefile

```diff
@@ -187,25 +187,25 @@ install: $(ELF)
 	@echo installing executable files to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@for e in ${EXECUTABLES}; do \
-		cp -f "$$e" ${DESTDIR}${PREFIX}/bin && \
+		cp -f "${.CURDIR}/$$e" ${DESTDIR}${PREFIX}/bin && \
```

Comment:
This doesn't make sense: how is this supposed to work on platforms which don’t set this weirdly named variable (and with all due respect to NetBSD, I am afraid, it is almost all of them)? Isn’t Makefile supposed to work as if `set -u` was set? Also, wouldn’t it be better to check whether `${.CURDIR}` is set and then just `cd` there?

Reply:
(For clarity: I mean `${.CURDIR}` is set on {Open,Net,Free}BSD's makes. Also I think the bug will affect all three of those platforms, when using base system `make` instead of GNU make, but I haven't tried.)